lib/pat.c

```diff
@@ -6227,6 +6227,57 @@ grn_pat_warm(grn_ctx *ctx, grn_pat *pat)
 int
```

Comment:
```suggestion
unsigned int
```

`return`している`table_size`の型が`unsigned int`なので、ここも`unsigned int`にしたほうがよいと思います。